csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/schema/Schema.scala

```diff
@@ -25,6 +25,8 @@ case class Quoted() extends GlobalDirective("quoted")
 
 case class TotalColumns(numberOfColumns: BigInt) extends GlobalDirective("totalColumns")
 
+case class IntegrityCheck(filepathColumn: String, includeFolder: Boolean) extends GlobalDirective("integrityCheck")
```

Comment:
Having global directives allows to have more modularity than command line option. For instance, if we want to validate 2 files (metadata and closure), we can enable it for one and not the other using directive within their schemas, this wouldn't be possible (or entail much more changes in the code that runs it) to do so.